truffle/com.oracle.truffle.api.dsl.test/src/com/oracle/truffle/api/dsl/test/interop/WriteNode.java

```diff
@@ -30,7 +30,7 @@
 
     @SuppressWarnings({"static-method", "unused"})
     @ExpectError({"The first argument must be a com.oracle.truffle.api.frame.VirtualFrame- but is java.lang.String"})
-    protected int access(String vf, Object receiver, Object name, Object value) {
+    protected int access(String frame, Object receiver, Object name, Object value) {
```

Comment:
Yes, although it's a "wrong frame parameter". Should we name that one "string" then?